src/perftest_parameters.c

```diff
@@ -2838,6 +2847,8 @@ void ctx_print_test_info(struct perftest_parameters *user_param)
 
 	if (user_param->post_list > 1)
 		printf(" Post List       : %d\n",user_param->post_list);
+	if (user_param->recv_post_list > 1)
+		printf(" Receive Post List : %d\n", user_param->recv_post_list);
```

Comment:
please change to " Recv Post List  : %d\n"

Reply:
Done